combat-trainer.lic

```diff
@@ -395,9 +395,10 @@ class LootProcess
   end
 
   def can_perform_training_ritual?(game_state)
+    return false unless DRStats.necromancer?
+    return false unless @ritual_type
     return false if game_state.necro_casting?
-    return false if @necro_train_first_aid && DRSkill.getxp('First Aid') > 32
-    return false if DRSkill.getxp('Thanatology') > 32
+    return false if DRSkill.getxp('First Aid') > 31 && DRSkill.getxp('Thanatology') > 31 && DRSkill.getxp('Skinning') > 31
```

Comment:
Based on a discussion with other players, it was decided to always do training rituals unless these three skills are locked

Reply:
Would the "cycle rituals" PR be good to get in first? It might make some of this logic easier

Reply:
Works for me

Reply:
#1629 is _almost_ done, are you up for finishing it?

Reply:
Yea of course, how should I do that?